helm/botkube/templates/servicemonitor.yaml

```diff
@@ -23,7 +23,6 @@ spec:
       helm.sh/chart: {{ include "botkube.chart" . }}
       app.kubernetes.io/instance: {{ .Release.Name }}
       app.kubernetes.io/managed-by: {{ .Release.Service }}
-      component: controller
```

Comment:
We can also add this on Service side, up to the reviewer 🙂

Reply:
IMO no need to do that 👍